docs/fundamentals/executor/executor-built-in-features.md

```diff
@@ -232,4 +232,33 @@ with (Flow().
         add(uses={'jtype': 'MyExecutor', 'metas': {'name': 'my-executor-1'}}).
         add(uses={'jtype': 'MyExecutor', 'metas': {'name': 'my-executor-2'}})) as f:
     f.post(on='/endpoint', inputs=DocumentArray([]), parameters={'param': 5, 'my-executor-1': {'param': 10}})
-```
\ No newline at end of file
+```
+
+
+````{admonition}  Note
+:class: caution
+
+As `parameters` does not have a fixed schema, it is declared with type `google.protobuf.Struct` in the `RequestProto`
+protobuf declaration. However, `google.protobuf.Struct` follows the JSON specification and does not 
+differentiate `int` from `float`. **So, data of type `int` in `parameters` will be casted to `float` when request is
+sent to executor.**
+
+As a result, users need be explicit and cast the data to the expected type as follows.
+
+```{code-block} python
+---
```

Comment:
You can wrap this inside 2 tabs (Do and Don't) as in here: https://docs.jina.ai/fundamentals/clean-code/
Source code: https://github.com/jina-ai/jina/blob/master/docs/fundamentals/clean-code.md line 13
In the don't tab, you can show what happens if you don't cast

Reply:
Same as above.